packages/theme/src/styles/global.ts

```diff
@@ -0,0 +1,26 @@
+import { globalCss } from "@stitches/react";
+
+export const globalStyles = globalCss({
```

Comment:
Just wanted to check if any of this is different than what's currently in the application? Will we need to fix any stylings in the application because it's not expecting any of these global styles?

Reply:
Good question. This is a slimmed down version of whats in the current [index.scss file](https://github.com/Localitos/Backend/blob/main/client/src/index.scss). I actually removed the margin and padding reset since thats a bit heavy handed and would cause some issues. The `fontSize: "100%",` is a safe addition.